lib/translation_maps/locations.rb

```diff
@@ -1,323 +1,325 @@
 {"anxa"=>"Forrestal Annex",
```

Comment:
I wouldn't include this in the application src. Better, I think, to have a hash pulled from  http://bibdata.princeton.edu/locations/holding_locations.json and stashed in a constant when the process starts each time--otherwise you're managing location data in yet another place.